azkaban-common/src/main/java/azkaban/jobExecutor/JavaProcessJob.java

```diff
@@ -48,6 +50,7 @@ public class JavaProcessJob extends ProcessJob {
   public static final String NATIVE_LIBRARY_PATH_PREFIX = "native.library.path.";
 
   public static String JAVA_COMMAND = "java";
+  public static String DEFAULT_JAVA11_COMMAND = "java11";
```

Comment:
probably rename to DEFAULT_JAVA11_BINARY _PATH?

Reply:
i also thought of this... But i think the default "java11" "java" are commands (shortcuts) but not the actual binary paths.